<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>calcForm</class>
 <widget class="QWidget" name="calcForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>calcForm</string>
  </property>
  <widget class="QTableView" name="tableView">
   <property name="geometry">
    <rect>
     <x>11</x>
     <y>340</y>
     <width>781</width>
     <height>250</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>250</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>250</height>
    </size>
   </property>
  </widget>
  <widget class="QTextEdit" name="txtThermoState">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>5</y>
     <width>401</width>
     <height>321</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>321</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>350</height>
    </size>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>361</width>
     <height>101</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="lblMode">
      <property name="text">
       <string>Input Mode: </string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QComboBox" name="cmbMode">
      <item>
       <property name="text">
        <string>Select Mode</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>State (p,T)</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>State (p,h)</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>State (p,s)</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>State (h,s)</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>P_Sat (T)</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>T_Sat (p)</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>State (Rho,T)</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>P3_Sat(h)</string>
       </property>
      </item>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="lblProperty1">
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="lnProperty1"/>
    </item>
    <item row="1" column="2">
     <widget class="QComboBox" name="cmbPropertyUnit1">
      <property name="font">
       <font>
        <pointsize>8</pointsize>
       </font>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="lblProperty2">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QLineEdit" name="lnProperty2"/>
    </item>
    <item row="2" column="2">
     <widget class="QComboBox" name="cmbPropertyUnit2">
      <property name="font">
       <font>
        <pointsize>8</pointsize>
       </font>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>300</y>
     <width>371</width>
     <height>25</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QPushButton" name="btnPlot">
      <property name="text">
       <string>Plot</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btnCalculate">
      <property name="text">
       <string>Calculate</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btnClose">
      <property name="text">
       <string>Close</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
